### PR TITLE
Fixed invalid parameter card_date. Added formatter for card payment data

### PR DESCRIPTION
**Pull request by ProkopovVitaliy**
Fixed invalid parameter card_date #39

### CODE REVIEW

File: card-gateway/Gateway/Request/Builder/CardPaymentBuilder.php [hunk 1]
```diff
@@ -31,8 +46,12 @@ public function build(array $buildSubject)
         return [
             RequestFields::CARD => $payment->getData(CardPaymentInterface::NUMBER),
             RequestFields::CARD_CVV => $payment->getData(CardPaymentInterface::CVV),
-            RequestFields::CARD_EXP_MONTH => $payment->getData(CardPaymentInterface::EXPIRATION_MONTH),
-            RequestFields::CARD_EXP_YEAR => $payment->getData(CardPaymentInterface::EXPIRATION_YEAR),
+            RequestFields::CARD_EXP_MONTH => $this->cardPaymentDataFormatter->format(
```
Comment (mcspronko) [inline card-gateway/Gateway/Request/Builder/CardPaymentBuilder.php]: Better to use method name to have formatter for a year and separately for a month.

File: card-gateway/Model/CardPaymentDataFormatter.php [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Copyright © Pronko Consulting (https://www.pronkoconsulting.com)
+ * See LICENSE for the license details.
+ */
+declare(strict_types=1);
+
+namespace Pronko\LiqPayCardGateway\Model;
```
Comment (mcspronko) [inline card-gateway/Model/CardPaymentDataFormatter.php]: Please move this class to the Pronko\LiqPayCardGateway\Gateway\Request directory.

File: card-gateway/Model/CardPaymentDataFormatter.php [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        return substr($value, -2);
+    }
+
+    /**
+     * @param string $value
+     * @return string
+     */
+    private function getFormattedMonth($value)
+    {
+        return strlen ($value) < 2 ? '0'. $value : $value;
```
Comment (mcspronko) [inline card-gateway/Model/CardPaymentDataFormatter.php]: space is missing before .

**Review by mcspronko — CHANGES_REQUESTED** [review]

File: card-gateway/Gateway/Request/Builder/CardPaymentBuilder.php (hunk 1 above)
Comment (ProkopovVitaliy) [reply card-gateway/Gateway/Request/Builder/CardPaymentBuilder.php]: @mcspronko something like `getFormattedYear($value)` and `getFormattedMonth($value`) ?

**Review by mcspronko — APPROVED** [review]

**Pull request merged**